src/interldpc.c

```diff
@@ -163,43 +163,12 @@ void qpsk_modulate_frame(COMP tx_symbols[], int codeword[], int n) {
     }
 }
 
-void interleaver_sync_state_machine(struct OFDM *ofdm,
-        struct LDPC *ldpc,
-        struct OFDM_CONFIG *config,
-        COMP codeword_symbols_de[],
-        float codeword_amps_de[],
-        float EsNo, int interleave_frames,
-        int *iter, int *parityCheckCount, int *Nerrs_coded) {
-    int coded_syms_per_frame = ldpc->coded_syms_per_frame;
-    int coded_bits_per_frame = ldpc->coded_bits_per_frame;
-    int data_bits_per_frame = ldpc->data_bits_per_frame;
-    float llr[coded_bits_per_frame];
-    uint8_t out_char[coded_bits_per_frame];
-    State next_sync_state_interleaver;
-
-    next_sync_state_interleaver = ofdm->sync_state_interleaver;
-
-    if ((ofdm->sync_state_interleaver == search) && (ofdm->frame_count >= (interleave_frames - 1))) {
-        symbols_to_llrs(llr, codeword_symbols_de, codeword_amps_de, EsNo, ofdm->mean_amp, coded_syms_per_frame);
-        iter[0] = run_ldpc_decoder(ldpc, out_char, llr, parityCheckCount);
-        Nerrs_coded[0] = data_bits_per_frame - parityCheckCount[0];
-
-        if ((Nerrs_coded[0] == 0) || (interleave_frames == 1)) {
-            /* sucessful decode! */
-            next_sync_state_interleaver = synced;
-            ofdm->frame_count_interleaver = interleave_frames;
-        }
-    }
-
-    ofdm->sync_state_interleaver = next_sync_state_interleaver;
-}
-
 /* measure uncoded (raw) bit errors over interleaver frame, note we
```

Comment:
Oh boy - nice to see this one go :+1:   Was so much trouble to get going, added a lot of complexity.  Oh well, that's experimental radio!

Reply:
In freedv_api,c the call just returns true:
```
int freedv_get_sync_interleaver(struct freedv *f) {
    return 1;
}
```